core/feconf.py

```diff
@@ -1504,3 +1505,41 @@ def get_empty_ratings() -> Dict[str, int]:
 COMMIT_TYPE_REVERT = 'revert'
 COMMIT_TYPE_EDIT = 'edit'
 COMMIT_TYPE_DELETE = 'delete'
+
+# The data type for the content in the TranslatableContent/TranslatedContent
```

Comment:
In order to avoid duplicate code in the domain and model layer.
Duplicate classes: TranslatedContentDict, TranslatableEntityType
Non duplicate classes: ContentInTranslatableContent, TranslatableContentFormat, TranslatableContentDict.

Nonduplicate classes can be relocated back into translation_domain but, is it fine to use feconf for 
TranslatedContentDict & TranslatableEntityType or they are good as earlier with a comment that after issue https://github.com/oppia/oppia/issues/14537 they will be fixed?

Reply:
Yeah, feconf is becoming a bit of a grab-bag. I would say, put the nonduplicate classes back in translation_domain but leave the duplicate ones here.

Reply:
Done.